cypher/cypher-docs/src/docs/graphgists/query-tuning/advanced-query-tuning-example.asciidoc

```diff
@@ -289,6 +289,7 @@ Predicates that will not work:
 * Several predicates combined using `OR` because the property type might differ between the predicates
 * Existence (eg. `WHERE exists(n.email)`) because no property type information is given
 
-Please note that if the predicate uses a parameter (eg. `WHERE n.prop > $param`) it can *not* trigger _index-backed order by_ in this version of Neo4j.
+[NOTE]
+If the predicate uses a parameter, such as `WHERE n.prop > $param`, it will not trigger _index-backed order by_ in this version of Neo4j.
 
 An existence constraint does not include any type information either and will thus not be enough to trigger _index-backed order by_.
```

Comment:
@SaschaPeukert should this paragraph be included in the admonition?

Reply:
Sure. Sounds useful